LayoutTests/http/tests/resourceLoadStatistics/exemptDomains/managed-domains-exempt-from-cookie-blocking-between-each-other-expected.txt

```diff
@@ -0,0 +1,33 @@
+Tests that managed domains are exempt from third-party cookie blocking between themselves.
```

Comment:
I agree this should work but we also intend to exempt other 3p domains, right? Or is it just between the domains in the list?